Projects/hCoreUI/Sources/Views/ContractRow.swift

```diff
@@ -165,7 +161,6 @@ private struct ContractRowButtonStyle: SwiftUI.ButtonStyle {
             background
         )
         .border(hBorderColor.primary, width: 0.5)
-        .colorScheme(.light)
         .clipShape(Squircle.default())
```

Comment:
Switch from Squircle to RounderRectangle. I am not sure about the corner radius amount.